transport/src/iosMain/kotlin/com/genesys/cloud/messenger/transport/network/PlatformSocket.kt

```diff
@@ -108,55 +124,49 @@ internal actual class PlatformSocket actual constructor(
                     handleErrorAndDeactivate(nsError, "Pong not received within interval [$pingInterval]")
                     return@scheduledTimerWithTimeInterval
                 }
-
-                log.i { "Sending ping" }
-                waitingOnPong = true
-                webSocket?.sendPingWithPongReceiveHandler { nsError ->
-                    waitingOnPong = false
-                    if (nsError != null) {
-                        handleErrorAndDeactivate(nsError, "Received pong error")
-                    } else {
-                        log.i { "Received pong" }
-                    }
-                }
+                sendPing()
             }
         }
     }
 
-    private fun cancelPings() {
-        pingTimer?.invalidate()
-        pingTimer = null
-        waitingOnPong = false
-    }
-
-    private fun listenMessages(listener: PlatformSocketListener) {
-        webSocket?.receiveMessageWithCompletionHandler { message, nsError ->
-            when {
-                nsError != null -> {
-                    handleErrorAndDeactivate(nsError, "Receive handler error")
-                    return@receiveMessageWithCompletionHandler
-                }
-                message != null -> {
-                    message.string?.let { listener.onMessage(it) }
-                }
+    private fun sendPing() {
+        log.i { "Sending ping" }
+        if(waitingOnPong) {
```

Comment:
<style nitpick> I think we typically have a space between the `if` and the opening `(`

Reply:
You are correct. Forgot to run the lint command before pushing.